app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new(params[:id])
```

Comment:
Wondering why you have `params[:id]` passed here.

Is this necessary? What does it do that `@user = User.new`. 💭

Reply:
Please address 😄